.github/CONTRIBUTING.md

```diff
@@ -1,112 +1,51 @@
- from a human being - submit a ticket via the helpdesk
-https://woocommerce.com/contact-us/ 
-
-**General usage and development questions**
-- WooCommerce Slack Community: https://woocommerce.com/community-slack/
-- WordPress.org Forums: https://wordpress.org/support/plugin/woocommerce
-- The WooCommerce Help and Share Facebook group
-
-**Customizations**
-- [WooExperts](https://woocommerce.com/experts/)
-- [Codeable](https://codeable.io/)
+- [Translating strings into your language](https://github.com/woocommerce/woocommerce/wiki/Translating-WooCommerce).
+- Answering questions on the various WooCommerce communities like the [WP.org support forums](https://wordpress.org/support/plugin/woocommerce/).
+- Testing open [issues](https://github.com/woocommerce/woocommerce/issues) or [pull requests](https://github.com/woocommerce/woocommerce/pulls) and sharing your findings in a comment.
+- Testing WooCommerce beta versions and release candidates. Those are announced in the [WooCommerce development blog](https://woocommerce.wordpress.com/).
+- Submitting fixes, improvements, and enhancements.
+- To disclose a security issue to our team, [please submit a report via HackerOne](https://hackerone.com/automattic/).
 
-## Build process 🛠
+If you wish to contribute code, before [forking](https://help.github.com/articles/fork-a-repo/) WooCommerce, commiting your changes,
+and [sendind a pull request](https://help.github.com/articles/using-pull-requests/), please see the information in the sections below.
 
-The source code found on GitHub does not contain compiled CSS or Javascript, and is missing some functionality such as Gutenberg Blocks and the REST API, until the build process is ran.
+WooCommerce is licensed under the GPLv2+, and all contributions to the project will be released under the same license. You maintain copyright over any contribution you make, and by submitting a pull request, you are agreeing to release that contribution under the GPLv2+ license.
 
-To run the build process, in a terminal go to your WooCommerce installation and run:
+If you have questions about the process to contribute code or want to discuss details of your contribution, you can contact WooCommerce core developers on the #core channel in the [WooCommerce community Slack](https://woocommerce.com/community-slack/).
 
-```bash
-npm install
-composer install
-npm run build
-```
+## Getting started
 
-To get your development environment running we recommend [reading here](https://github.com/woocommerce/woocommerce/wiki/How-to-set-up-WooCommerce-development-environment).
+- [How to set up WooCommerce development environment](How-to-set-up-WooCommerce-development-environment)
+- [Minification of SCSS and JS](Minification-of-SCSS-and-JS)
+- [String localisation guidelines](String-localisation-guidelines)
+- [Running unit tests](https://github.com/woocommerce/woocommerce/blob/master/tests/README.md)
+- [Running e2e tests](End-to-end-Testing)
```

Comment:
Could be worth mentioning also our approach to GitHub branches, if to use merge or rebase to keep the branch up to date. What do you reckon?

Reply:
I added a link to the Git Flow wiki page. This should cover our approach to branches. Regarding merging or rebasing, I don't have a strong opinion. I usually merge instead of rebasing. Maybe we should discuss this with the rest of the team and then update the Git Flow wiki page?

Reply:
I agree with the approach.

Reply:
Rebasing requires more instructions/guardrails, so recommending merging should be easier to wrangle :)